content/docs/insights/get-started/begin.md

```diff
@@ -0,0 +1,191 @@
+weight: 2
+menu:
+  insights:
+    parent: insights-get-started
+    identifier: insights-get-started-begin
+    weight: 2
+---
+
+## Before you begin
+
+First, let's run through a few prerequisites and quick steps to ensure you ready to create your first Account Discovery scan.
+
+- Ensure you’re an admin of your Pulumi organization.
+- Verify you have permissions to create credentials in the provider account you want to scan.
+- You're using Pulumi's **Team**, **Enterprise**, or **Business Critical** edition.
+
+If you're new to Pulumi you can click here to [start a free trial](https://app.pulumi.com/?create-organization).
+
+## Create an ESC Environment
+
+Pulumi Insights Account Discovery requires read-only access to your cloud accounts. This access is granted by [creating an ESC environment](/docs/esc/get-started/create-environment/) that generates valid credentials for the corresponding Pulumi provider when accessed.
+
+{{% notes "info" %}}
+Account Discovery leverages Pulumi ESC to securely manage the credentials required to discover and read infrastructure resources, aligning with enterprise best practices for managing application secrets.
+{{% /notes %}}
+
+To create an environment, [sign into the Pulumi cloud](https://app.pulumi.com/) console and navigate to **Pulumi ESC** and select **Environments** in the left-hand menu.
+
+Next, click **Create Environment** and enter a name for the project and environment, such as `insights-discovery-project` and `insights-environment` and then click **Create**.
+
+Leave the default environment definition for now, and you will return to finish configuring ESC after you create the required credentials.
+
+## Create and configure cloud credentials
+
+<!-- TODO: get oracle cloud chooser working -->
+
+{{% chooser cloud "aws,azure,oracle,kubernetes" %}}
+
+{{% choosable cloud aws %}}
+
+To configure Pulumi Insights with AWS, you will use [OpenID Connect (OIDC)](docs/pulumi-cloud/access-management/oidc/) for authentication. Follow these steps:
+
+1. Log in to the [AWS Management Console](https://console.aws.amazon.com/iam/).
+2. Go to the **Roles** section and create a new role.
+3. Select the **Web identity** trusted entity type and choose `api.pulumi.com/oidc` as your identity provider.
+4. Select the name of your Pulumi organization under **Audience**.
+5. Filter to the `ReadOnlyAccess` policy name.
+6. Click **Create**.
+
+This will set up a trust relationship to allow Pulumi Cloud to assume the role using the following trust policy:
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": "sts:AssumeRoleWithWebIdentity",
+            "Principal": {
+                "Federated": "arn:aws:iam::<ACCOUNT_ID>:oidc-provider/api.pulumi.com/oidc"
+            },
+            "Condition": {
+                "StringEquals": {
+                    "api.pulumi.com/oidc:aud": [
+                        "pulumi"
```

Comment:
maybe I am wrong here and the org isnt needed. not sure now. I know we set ours up with `pulumi:aws`

Reply:
adjusted based on my demo account setup, thx!